docs/gh-build-and-sign.yml

```diff
@@ -86,9 +85,9 @@ jobs:
         --publisher-name "Contoso"
         --description "One Sign CLI demo"
         --description-url "https://github.com/dotnet/sign"
-        --azure-key-vault-managed-identity true
-        --azure-key-vault-url "${{ secrets.KEY_VAULT_URL }}"
-        --azure-key-vault-certificate "${{ secrets.KEY_VAULT_CERTIFICATE_ID }}"
+        --azure-credential-type "azure-cli"
+        --azure-key-vault-url "${{ secrets.KEY_VAULT_URL }}" # This does not need to be a secret and is just a placeholder
+        --azure-key-vault-certificate "${{ secrets.KEY_VAULT_CERTIFICATE_ID }}" # This does not need to be a secret and is just a placeholder
```

Comment:
Just curious. If it's a placeholder, then where is the actual value coming from in this example? An environment variable?

Reply:
In my use case I hard-coded the values as they aren't really secrets. The value will need to be figured out by whoever is hosting the KV.

Reply:
Thanks

Reply:
GitHub Actions secrets can be managed through the GitHub Actions web page.  `secrets` is predefined.  See https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/use-secrets?tool=webui